store/store_test.go

```diff
@@ -2091,7 +2091,8 @@ func (s *storeTestSuite) TestDetails(c *C) {
 	}
 	result, err := sto.SnapInfo(spec, nil)
 	c.Assert(err, IsNil)
-	c.Check(result.Name(), Equals, "hello-world")
+	// TODO parallel-install: use of proper instance/store name
+	c.Check(result.InstanceName(), Equals, "hello-world")
```

Comment:
assuming s/InstanceName/Name/ later this is just fine, there's a question whether SnapInfo should take foo_bar names, I don't think it needs to, but we need some convention

Reply:
Perhaps we should establish a convention that all store APIs (as in coming from the `store` package) take `*_<key>` names. Not passing the full name here would complicate the callers and the structures we pass around, eg. CurrentSnap would need to have some unique ID/index set by the caller, SnapAction would then need to reference back to the current snaps in case of refresh and so on.